app/actions/todos.go

```diff
@@ -20,3 +22,30 @@ func Index(c buffalo.Context) error {
 
 	return c.Render(http.StatusOK, r.HTML("/todos/index.plush.html"))
 }
+
+func NewTask(c buffalo.Context) error {
+	todo := models.Todo{}
+	c.Set("todo", todo)
+
+	return c.Render(http.StatusOK, r.HTML("todos/new.plush.html"))
+}
+
+func PostNewTask(c buffalo.Context) error {
+	u, _ := uuid.NewV4()
+	todo := models.Todo{ID: u}
+
+	if err := c.Bind(&todo); err != nil {
+		return err
+	}
+
+	tx := c.Value("tx").(*pop.Connection)
+
+	err := tx.Create(&todo)
+	if err != nil {
+		return errors.WithStack(err)
+	}
+
+	c.Set("todo", todo)
+
+	return c.Render(http.StatusOK, r.HTML("todos/new.plush.html"))
```

Comment:
It is our convention that, after we create the resource, we redirect to the resource listing.
Let's redirect to the todos listing. e.g:
`c.Redirect(http.StatusSeeOther, "/"`